scripts/wikipedia_covid/timeseries/Config.py

```diff
@@ -0,0 +1,1155 @@
+# NOTE: Keep all lower case.
+
+replace_headers = {
+    "active cases": "active_cases", # Get rid of spaces
+    "active cases and recoveries": "cases", # See Connecticut, USA
```

Comment:
I'm not sure the "See" comments are helpful-- expand or remove?

Reply:
done